src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,112 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, driver_license) VALUES (?, ?)";

```

Comment:
If you remember, Bohdan said that a rule of good tone is to add semicolon at the end of your query.

Reply:
I remember, but then the mentor said it's better not to do that. let's see how it will be this time))

Reply:
both options are good, but without `;` it reads easier, since you have kinda `);";` and it is very noticeable